libyul/optimiser/SSAValueTracker.cpp

```diff
@@ -33,12 +33,18 @@ void SSAValueTracker::operator()(Assignment const& _assignment)
 		m_values.erase(var.name);
 }
 
+void SSAValueTracker::operator()(FunctionDefinition const& _funDef)
+{
+	for (auto const& var: _funDef.returnVariables)
+		setValue(var.name, nullptr);
```

Comment:
Didn't the other PR set to `zero` instead of `nullptr`?

Reply:
Take a look at the implementation of `setValue` - it uses a special zero expression if the second argument is nullptr.